_includes/markdown/Markup.md

```diff
@@ -285,10 +285,96 @@ Combining automated and manual testing practices allows 10up to maintain a high
+
+<h2 id="svg" class="anchor-heading">SVG {% include Util/top %}</h2>
+<abbr title="Scaleable Vector Graphics">SVG</abbr> has become a prevalent means for displaying rich vector graphics. At 10up we prefer to use SVGs over icon fonts for several reasons:
+
+1. Sharper in fidelity - icon fonts can be a bit pixelated, because the browser treats them as text (i.e. anti-aliased). While this can sometimes be overcome by adjusting `font-smoothing`. This approach is not recommended.
+2. Broader CSS control - individual stroke and fill properties can be adjusted.
+3. Semantics - An `<svg>` element has semantic meaning as an image. This helps communicate to browsers and assitive technology the intended meaning of the element.
+4. Accessibility - while font icons _can_ be implemented in an accessible way. There are obscure instances where glyphs have mixed Unicode meaning and can be output in inconsistent manner. Whereas, an SVG is output _as is_.
+
+### Best Practices &amp; Common Usage
+
+#### SVG icon in a button with text
+
+```
+<button type="button">
+    Menu
+    <svg viewBox="0 0 10 10"
+        class="svg-icon"
+        aria-hidden="true"
+        focusable="false">
+        <path d="m1 7h8v2h-8zm0-3h8v2h-8zm0-3h8v2h-8z"/>
+    </svg>
+</button>
+```
+
+Since the SVG is merely decorative and the "Menu" textually explains the button's purpose. Then we can leverage `aria-hidden="true"` and `focusable="false"` to hide the SVG-as-decorative-element from the accessibility tree.
+
+#### SVG icon in a button with _hidden_ text
+
+```
+<button type="button">
+    <svg class="svg-icon"
+        height="10"
+        width="10"
+        viewBox="0 0 10 10"
+        aria-hidden="true"
+        focusable="false">
+        <path d="m1 7h8v2h-8zm0-3h8v2h-8zm0-3h8v2h-8z"/>
+    </svg>
+    <span class="visually-hidden">
+        Menu
+    </span>
+</button>
+```
+
+The `<span>Menu</span>` still provides assistive technology with a textual explanation, while hiding the text from the viewer.
+
+The corresponding CSS for this:
+
+```
+.visually-hidden {
+    clip: rect(0, 0, 0, 0); // Clip the box to zero pixels.
+    height: 1px;
+    overflow: hidden; // Hide any overflowing elements or text.
+    position: absolute; // Move the text out of the flow of the container.
+    white-space: nowrap; // Text won't wrap to a second line.
+    width: 1px; // Reduce its height and width to just one pixel.
+}
+```
+
+#### SVG as non-decorative element
+
+If you would like to use an inline SVG. Here are some considerations to improve experience for assistive technology:
+
+* Use a descriptive `<title>` within the SVG.
+* Assign `role="image"`
```

Comment:
@colorful-tones above, you mentioned that the semantics of an SVG imply images, but here we're adding `role="image"`, which corrects semantics. Which is correct?